gatsby-browser.js

```diff
@@ -36,6 +36,11 @@ export const shouldUpdateScroll = (ctx) => {
     return false;
   }
 
+  if (location.pathname.match(/\/register\/form/)) {
+    window.scroll({ top: document.getElementById('su-gg-embed')?.offsetTop });
```

Comment:
```suggestion
    window.scroll({ top: document.getElementById('su-gg-embed').offsetTop });
```

It is not good practice to put the optional chain there. If you are expecting to scroll to something you should check to see if it exists first instead of defaulting to null/undefined here.

Reply:
Removed optional chaining and shifted that into a conditional check 😎 hopefully that resolves the localhost gatsby-browser error that @jdwjdwjdw had encountered.